src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,7 +5,43 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int bladeLength;
+    private int bladeMass;
+    private int enginePower;
```

Comment:
Access modifiers for fields should be private to encapsulate the data. It's good practice to provide access to fields via getters and setters, which you have done, but the fields themselves should be explicitly marked as private.